### PR TITLE
fix!: Make it work.

### PR DESCRIPTION
**Pull request by tupoy-ya**
- [ ] Fix crash when spawning old jsons.
- [x] Add godmode check.

### CODE REVIEW

**Comment by tupoy-ya** [comment]
Crash only occurs if spawning old jsons

File: BigBaseV2/src/services/vehicle/persist_car_service.cpp [hunk 1]
```diff
@@ -148,14 +150,20 @@ namespace big
 		}
 		if (!vehicle_json[is_visible_key].is_null())
```
Comment (tupoy-ya) [inline BigBaseV2/src/services/vehicle/persist_car_service.cpp:151]: This crashes for some reason.

**Comment by tupoy-ya** [comment]
I don't think i can fix this crash because it happens in model_attachment.

**Comment by TheGreenBandit** [comment]
It cannot be fixed, you would have to save load old jsons with old menu then save em with this commit menu

**Pull request merged**